cmd/kube-controller-manager/app/core.go

```diff
@@ -395,24 +395,18 @@ func startGarbageCollectorController(ctx ControllerContext) (bool, error) {
 }
 
 func startPVCProtectionController(ctx ControllerContext) (bool, error) {
-	if utilfeature.DefaultFeatureGate.Enabled(features.StorageObjectInUseProtection) {
```

Comment:
We don't typically remove the gate to go GA, we flip the feature status, and that's it.